### ISSUE TITLE
Lost connection with Qbittorrent

### ISSUE THREAD

**Issue by unuspromulti** [Radarr/Radarr#1264]
Received error msg and noticed link had gone down. Noticed last torrent had gone way past its seed limit, deleted it from Qbittorrent and problem has gone away, retested connection and now works again. 

Error:
Unable to communicate with Qbittorrent. JSON integer 18446744073709400000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.

Version
0.2.0.535

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709398000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:37pm

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:37pm

NzbDroneErrorPipeline
Invalid request Validation failed: -- Failed to get the list of torrents: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:36pm

QBittorrent
JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:36pm

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709398000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:36pm

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:36pm

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709400000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:35pm

DownloadMonitoringService
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:35pm

DownloadClientCheck
Unable to communicate with Qbittorrent.: JSON integer 18446744073709400000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
2:34pm

Details
Message 
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.

Exception 
Newtonsoft.Json.JsonReaderException: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
   at Newtonsoft.Json.JsonTextReader.ParseNumber()
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.ReadInternal()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at NzbDrone.Common.Serializer.Json.Deserialize[T](String json) in C:\projects\radarr-usby1\src\NzbDrone.Common\Serializer\Json.cs:line 37
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxy.ProcessRequest[TResult](HttpRequestBuilder requestBuilder, QBittorrentSettings settings) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrentProxy.cs:line 184
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxy.GetTorrents(QBittorrentSettings settings) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrentProxy.cs:line 64
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.GetItems() in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrent.cs:line 78
   at NzbDrone.Core.Download.TrackedDownloads.DownloadMonitoringService.ProcessClientDownloads(IDownloadClient downloadClient) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TrackedDownloads\DownloadMonitoringService.cs:line 86
Close 

Radarr Ver. 0.2.0.535
develop 

×
Details
Message 
Unable to retrieve queue and history items from Qbittorrent: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.

Exception 
Newtonsoft.Json.JsonReaderException: JSON integer 18446744073709402000 is too large or small for an Int64. Path '[7].eta', line 1, position 5500.
   at Newtonsoft.Json.JsonTextReader.ParseNumber()
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.ReadInternal()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at NzbDrone.Common.Serializer.Json.Deserialize[T](String json) in C:\projects\radarr-usby1\src\NzbDrone.Common\Serializer\Json.cs:line 37
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxy.ProcessRequest[TResult](HttpRequestBuilder requestBuilder, QBittorrentSettings settings) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrentProxy.cs:line 184
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxy.GetTorrents(QBittorrentSettings settings) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrentProxy.cs:line 64
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.GetItems() in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrent.cs:line 78
   at NzbDrone.Core.Download.TrackedDownloads.DownloadMonitoringService.ProcessClientDownloads(IDownloadClient downloadClient) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TrackedDownloads\DownloadMonitoringService.cs:line 86
Close 

Radarr Ver. 0.2.0.535
develop

**Comment by onedr0p** [comment]
If ETA is in seconds, that's a long ass time:

https://www.google.com/search?q=18446744073709398000+int64&oq=18446744073709398000+int64&aqs=chrome..69i57.6127j0j7&sourceid=chrome&ie=UTF-8#safe=off&q=18446744073709402000+seconds+in+days&*

**Comment by skidaddl3** [comment]
I keep getting this same connection error.

I try the connection Test in Rararr Settings>Download Client>qBittorrent (Connection Edit Page)>Test when I notice the error in the health status of the Radarr system.
When it failed, in qBittorrent I found and I changed the state of all the torrents I had above my seeding cut off of 0.10 from "Forced Resume" to just "Resume" for Radarr to continue it's moving/cleaning process (I like to force seed, as the torrent tends to stall more when it is not forced). Went back to Settings>Download Client>qBittorrent (Connection Edit Page)>Test and it said successful. I then re-test health and goes back to normal for a while.

I haven't confirmed if it is this exact problem/caveat.
